package.json

```diff
@@ -139,14 +133,15 @@
     "eslint-plugin-spellcheck": "^0.0.17",
     "execa": "^5.1.1",
     "glob": "^7.2.0",
-    "husky": "^3.1.0",
+    "husky": ">=6",
```

Comment:
Why upgrade husky in this PR?

Also, I think using `>=` is too permissive. We don't know that husky won't make breaking changes that will negatively affect our ability to run git hooks.

Reply:
Good catch. It was automatically done by 

I just took the chance and upgraded it to the newest version 293ce82c0824dc4117c26fd96be8857f0352a238